config/config_exec/browserstack_conf/local.conf.js

```diff
@@ -21,18 +15,17 @@ exports.config = {
         env:url,
     },
     capabilities: {
-        'browserstack.user': 'truptigarotkar1',
-        'browserstack.key': 'FotasUZMqVHy848ZhqxE',
-        'browserstack.local': true,
-        'browserstack.debug': true,
-        'browserName': 'firefox',
+      'browserstack.user': credential.browserstack.username,
+      'browserstack.key': credential.browserstack.key,
+      'browserstack.local': true,
+      'browserstack.debug': true,
+      'browserName': 'firefox',
       },
       
- // Code to start browserstack local before start of test
- beforeLaunch: function(){
+beforeLaunch: function(){
   console.log("Connecting local");
   return new Promise(function(resolve, reject){
-    bs_local.start({'key': 'FotasUZMqVHy848ZhqxE','proxy-host': 'internetpu', 'proxy-port': '8085' }, function(error) {
+    bs_local.start({'key': exports.config.capabilities['browserstack.key'],'proxy-host': credential.proxy.host, 'proxy-port': credential.proxy.port }, function(error) {
```

Comment:
We don't necessarily need this proxy when we run in Jenkins or any other platform, so think of a solution where we get this on demand

Reply:
Done, now it is on demand